CHANGELOG.md

```diff
@@ -9,6 +9,34 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.2.0 - 2019-12-18
```

Comment:
pedant note: do we keep it 2019-12-18 or shall we update it to 2019-12-19?

Reply:
will update